### PR TITLE
🌱  Clusterctl support for VMware Cloud Director as Infrastructure Provider

### PR DESCRIPTION
**Pull request by sahithi**
**What this PR does / why we need it**:
Adds clusterctl support for VMware Cloud Director as Infrastructure Provider - https://github.com/vmware/cluster-api-provider-cloud-director

### CODE REVIEW

**Comment by enxebre** [comment]
Thanks!
Could you please squash commits into a single one? Also can you please make sure to fix EasyCLA?

File: cmd/clusterctl/client/config/providers_client.go [hunk 1]
```diff
@@ -35,26 +35,27 @@ const (
 
 // Infra providers.
 const (
-	AWSProviderName        = "aws"
```
Comment (killianmuldoon) [inline cmd/clusterctl/client/config/providers_client.go:38]: Do you know why all of these lines were replaced in this change?

**Comment by killianmuldoon** [comment]
/lgtm 

Pending squash/CLA

**Comment by sahithi** [comment]
@killianmuldoon @enxebre 
Can you please review and approve it? I addressed all the comments.

**Comment by enxebre** [comment]
thanks!
/approve

**Pull request merged**